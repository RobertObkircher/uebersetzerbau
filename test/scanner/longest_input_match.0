end39 test 39end
