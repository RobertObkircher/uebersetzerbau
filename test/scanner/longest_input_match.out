id end39
id test
num 39
end
